### ISSUE TITLE
Tekkit Legends error "Please make sure you are using the server version of the FTB modpack!"

### ISSUE THREAD

**Issue by darioschmid** [itzg/docker-minecraft-server#1461]
### Describe the problem

I try to run a Tekkit Legends Server, but I can't get it to run. I get the error "Please make sure you are using the server version of the FTB modpack!"

Things I've already tried:
- Set Version to 1.7.10 (Minecraft Version Tekkit Legends runs on)
- Set Ftb Legacy Java Fixer to true
- used both :latest and :java8
- Disabled Health Check
- Omitting the volume mount
- Set Use Modpack Start Script to false

When downloading the https://servers.technicpack.net/Technic/servers/tekkit-legends/Tekkit_Legends_Server_v1.1.1.zip file and starting the server manually (without using docker), it runs fine. So the error shouldn't be the TechnicPack's fault.

### Container definition

version: '3.3'
services:
    itzg-tekkit-legends:
        image: itzg/minecraft-server:java8
        container_name: itzg-tekkit-legends
        environment:
            - EULA=TRUE
            - TYPE=FTB
            - 'FTB_SERVER_MOD=https://servers.technicpack.net/Technic/servers/tekkit-legends/Tekkit_Legends_Sends_Server_v1.1.1.zip'
        restart: unless-stopped
        ports:
            - '25567:25565'
        volumes:
            - /srv/minecraft/itzg-tekkit-legends:/data

### Container logs

[init] Changing ownership of /data to 1000 ...
[init] Running as uid=1000 gid=1000 with /data as 'drwxrwxr-x 2 1000 1000 4096 Apr  9 14:31 /data'
[init] Resolved version given LATEST into 1.18.2 and major version 1.18
[init] Resolving type given FTB
[init] Looking for Feed-The-Beast / CurseForge server modpack.
[init] Downloading modpack from https://servers.technicpack.net/Technic/servers/tekkit-legends/Tekkit_Legends_Server_v1.1.1.zip...
[init] Unpacking FTB server modpack /data/Tekkit_Legends_Server_v1.1.1.zip ...
...........................................................................................................................................................................................................................................................................................................................................................................................................
[init] Please make sure you are using the server version of the FTB modpack!

**Comment by itzg** [comment]
Sorry for the delay in responding. 

The modpack handling is currently unable to detect and handle some modpack layouts and this appears to be one of them. I plan on overhauling the modpack support and will include this one as a test case. 

In the meantime, you can try running this with the zip referenced from `GENERIC_PACK`, `TYPE=FORGE`, and `VERION` set appropriately.

**Comment by maludo99** [comment]
I found the reason it wasn't working! When trying to start the server, the docker image is looking for start scripts called "ServerStart.sh", "serverstart.sh", "ServerStartLinux.sh", etc., but for Tekkit Legends the start script is just called "start.sh" which isn't searched for.
For a hotfix you can just **rename "start.sh" to "serverstart.sh"**, but adding "start.sh" as a valid start script name is obviously the better option. Unfortulately, I couldn't create a branch and merge request in order to fix it myself.

**TLDR; rename "start.sh" to "serverstart.sh"**

**Comment by itzg** [comment]
Great find. That's an easy change I make in the image's script.

**Comment by fpatrick** [comment]
> "start.sh" to "serverstart.sh"

In the last  2 hours I have been reading issues and the documentation, trying to run https://www.curseforge.com/minecraft/modpacks/better-minecraft-fabric without success.

Changing start.sh to serverstart.sh was enough to run the forge version using the generic_pack type. Fabric version failed in any way I tried, always missing something.

**Comment by itzg** [comment]
@fpatrick That's a different modpack. Open a new issue and provide full container logs and configs.
